lib/wda/webdriveragent.js

```diff
@@ -333,12 +333,7 @@ class WebDriverAgent {
   }
 
   set fullyStarted (started = false) {
```

Comment:
do we still need the `fullyStarted` concept?

Reply:
What was the purpose of this originally?

Reply:
@imurchie has to know ;)